projects/canopy/src/lib/carousel/carousel.component.scss

```diff
@@ -1,5 +1,61 @@
+@import '../../styles/mixins.scss';
+
 .lg-carousel {
   overflow: hidden;
+  position: relative;
+
+  &__autoplay {
+    position: absolute;
+    right: var(--space-sm);
+    z-index: 1;
```

Comment:
Could you add this to the z-index list in the main variable file please (make sure you rebase).